.github/workflows/build-and-test.yml

```diff
@@ -2921,7 +2934,7 @@ jobs:
       - name: "Install chroot"
         uses: ./.github/actions/multistrap
         with:
-          arch: armv7
+          multistrap-config: ${{ matrix.multistrap-config }}
```

Comment:
I was convinced `arch` there was used by some of the customs actions?

Reply:
https://github.com/coqui-ai/STT/blob/0652121b834162ee48c063eee9c124bb7ed29893/.github/actions/multistrap/action.yml#L4

Reply:
So all that logic needs to live on the `multistrap` action side ...

Reply:
Using the correct config file is already done on the `multistrap` action side, you should stick with `arch` being there and move the handling of new platforms there.

Reply:
Fixed